design-proposals/klip-55-tablespaces.md

```diff
@@ -1,3 +1,210 @@
 # KLIP 55 - Tablespaces
 
-Just a placeholder. :-)
+**Author**: @MichaelDrogalis | 
+**Release Target**: TBD | 
+**Status**: In Discussion | 
+**Discussion**: _to be added_
+
+Tablespaces help you use knowledge of the usage pattern of a set of ksqlDB objects to optimize performance.
```

Comment:
they also provide a useful hook from which to hang various other things in future e.g. aggregated stats, security permissions